### DISCUSSION TITLE
Upgrading the hard drive that has dietpi_userdata

### DISCUSSION THREAD

**Discussion by Psych85** [MichaIng/DietPi#6039] [Q&A] [answered]
I currently have dietpi_userdata pointing to a 80gb hard drive. I want to upgrade this to a larger drive and wondered what is the best way to do this? Perhaps this question has been asked before but I could not find a discussion on it. Any suggestions welcome.

**Comment by Joulinar** [comment, marked as answer]
Attached the second drive and copy all data over (just copy, not move 😉 ). Once done, simply recreate the symbolic link for the user data pointing the new drive.

Ah easier, open drive manager and select the new drive as home for user data. This should do everything automatically.

**Comment by MichaIng** [comment]
If there is no data stored on it, I suggest to also reformat it (can be also done with `dietpi-drive_manager`. Use ext4, or probably F2FS if it's an SSD.

**Reply by Psych85** [reply]
Thanks for the suggestions. I was having some power issues as I was using a Rpi3 and trying to copy over data and I could hear the drive clicking and failing to read and write. I shutdown the server and copied over the files using a laptop and reconnected the new drive whilst editing fstab and changing the symbolic link. All sorted now. Thanks guys :)
